search_service/models/index_map.py

```diff
@@ -1,131 +1,8 @@
-import textwrap
-
-# todo: Move it to amundsen common repo
-DEFAULT_INDEX_MAP = textwrap.dedent("""
-{
-"mappings":{
-    "table":{
-      "properties": {
-        "name": {
-          "type":"text",
-          "analyzer": "simple",
-          "fields": {
-            "raw": {
-              "type": "keyword"
-            }
-          }
-        },
-        "schema": {
-          "type":"text",
-          "analyzer": "simple",
-          "fields": {
-            "raw": {
-              "type": "keyword"
-            }
-          }
-        },
-        "last_updated_timestamp": {
-          "type": "date",
-          "format": "epoch_second"
-        },
-        "description": {
-          "type": "text",
-          "analyzer": "simple"
-        },
-        "column_names": {
-          "type":"text",
-          "analyzer": "simple",
-          "fields": {
-            "raw": {
-              "type": "keyword"
-            }
-          }
-        },
-        "column_descriptions": {
-          "type": "text",
-          "analyzer": "simple"
-        },
-        "tags": {
-          "type": "keyword"
-        },
-        "cluster": {
-          "type": "text"
-        },
-        "database": {
-          "type": "text"
-        },
-        "key": {
-          "type": "keyword"
-        },
-        "total_usage":{
-          "type": "long"
-        },
-        "unique_usage": {
-          "type": "long"
-        }
-      }
-    }
-  }
-}""")
-
-USER_INDEX_MAP = textwrap.dedent("""
-{
-"mappings":{
-    "user":{
-      "properties": {
-        "email": {
-          "type":"text",
-          "analyzer": "simple",
-          "fields": {
-            "raw": {
-              "type": "keyword"
-            }
-          }
-        },
-        "first_name": {
-          "type":"text",
-          "analyzer": "simple",
-          "fields": {
-            "raw": {
-              "type": "keyword"
-            }
-          }
-        },
-        "last_name": {
-          "type":"text",
-          "analyzer": "simple",
-          "fields": {
-            "raw": {
-              "type": "keyword"
-            }
-          }
-        },
-        "name": {
-          "type":"text",
-          "analyzer": "simple",
-          "fields": {
-            "raw": {
-              "type": "keyword"
-            }
-          }
-        },
-        "total_read":{
-          "type": "long"
-        },
-        "total_own": {
-          "type": "long"
-        },
-        "total_follow": {
-          "type": "long"
-        }
-      }
-    }
-  }
-}""")
+from amundsen_common.models.index_map import TABLE_INDEX_MAP
 
 
 class IndexMap:
-    def __init__(self, map: str = DEFAULT_INDEX_MAP) -> None:
+    def __init__(self, map: str = TABLE_INDEX_MAP) -> None:
```

Comment:
given you move the index file to common, do we still need this file here?

Reply:
it could be moved to amundsencommon - but i think it's needed in some form right?

Reply:
you're right we dont - i removed it